### PR TITLE
Added getFrameCount method to VideoCapture

### PR DESCRIPTION
**Pull request by sirotenko**
When working with the video it is very important to know its size. This simple pull request add this functionality.

### CODE REVIEW

File: src/VideoCaptureWrap.cc [hunk 1]
```diff
@@ -33,6 +33,7 @@ void VideoCaptureWrap::Init(Local<Object> target) {
   Nan::SetPrototypeMethod(ctor, "setHeight", SetHeight);
   Nan::SetPrototypeMethod(ctor, "setPosition", SetPosition);
   Nan::SetPrototypeMethod(ctor, "getFrameAt", GetFrameAt);
+  Nan::SetPrototypeMethod(ctor, "nFrames", NumFrames);
```
Comment (mvines) [inline src/VideoCaptureWrap.cc]: How about making this method name more closely match the underlying CV property.  I suggest "getFrameCount".

**Comment by mvines** [comment]
lgtm, we need @peterbraden to merge though :)

**Comment by peterbraden** [comment]
LGTM, Thanks!

**Comment by peterbraden** [comment]
(BTW @mvines I added you as a collaborator so you don't have to wait for me on things like this :) )

**Pull request merged**